jobs/cjd_sample/pom.xml

```diff
@@ -8,12 +8,25 @@
   <version>1.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
+  <!--
+    The parent pom defines common style checks and testing strategies for our samples.
+    Removing or replacing it should not affect the execution of the samples in anyway.
+  -->
+  <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>LATEST</version>
```

Comment:
We want to be able to control when the sample is updated to match the parent releases, so please change LATEST to 1.0.9